js/apps/admin-ui/src/realm-settings/RealmSettingsTabs.tsx

```diff
@@ -350,50 +351,60 @@ export const RealmSettingsTabs = ({
           >
             <RealmSettingsTokensTab save={save} realm={realm} />
           </Tab>
-          <Tab
-            title={
-              <TabTitleText>{t("realm-settings:clientPolicies")}</TabTitleText>
-            }
-            data-testid="rs-clientPolicies-tab"
-            {...clientPoliciesTab}
-          >
-            <RoutableTabs
-              mountOnEnter
-              defaultLocation={toClientPolicies({
-                realm: realmName,
-                tab: "profiles",
-              })}
+          {!useServerInfo().profileInfo?.disabledFeatures?.includes(
```

Comment:
It's best to call hooks on top and store the value it returns as it prevents accidentally violating the [rules of hooks](https://legacy.reactjs.org/docs/hooks-rules.html) if a condition if ever added to the component.